Error: Cannot find module '../../../../graphql-codegen-core/dist'

npm -g ls graphql-code-generator
/usr/local/lib
└── graphql-code-generator@0.10.7

npm -g ls graphql
/usr/local/lib
└── graphql@0.13.2

npm -g ls graphql-codegen-typescript-template
/usr/local/lib
└── graphql-codegen-typescript-template@0.10.7

gql-gen -s http://localhost:3000/graphql --template ts --out ./src/typings/index.ts
internal/modules/cjs/loader.js:583
    throw err;
    ^

Error: Cannot find module '../../../../graphql-codegen-core/dist'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/graphql-code-generator/dist/loaders/schema/schema-from-export.js:42:14)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)

--- a/package.json
+++ b/package.json
@@ -11,8 +11,9 @@
     "test": "lerna run test",
     "lint": "lerna run lint",
     "precommit": "lint-staged",
-    "prerelease": "yarn build && yarn test",
-    "prerelease:canary": "yarn build && yarn test",
+    "test-and-build": "yarn test && yarn build",
+    "prerelease": "yarn test-and-build",
+    "prerelease:canary": "yarn test-and-build",
     "release": "lerna publish --exact --force-publish=\"*\"",
     "release:canary": "lerna publish --force-publish=\"*\" --canary --exact",
     "generate:examples": "./dev-test/generate-all.sh"

